apps/modernization-ui/src/apps/patient/add/generalInformation/generalInformation.tsx

```diff
@@ -27,6 +24,7 @@ export default function GeneralInformation({ id, title }: { id?: string; title?:
                             render={({ field: { onChange, value, name }, fieldState: { error } }) => (
                                 <DatePickerInput
                                     defaultValue={value}
+                                    label="Information as of Date"
```

Comment:
for the [issue](issue: The required property needs to be added to DatePickerInput since the label was removed)

I dont think the label should be added as a required property because in the demographics tab we are using it differently in the bottom cards like Sex and Birth, ethnicity. I tried using flexbox prop but the styles are changing the behavior of the UI

Reply:
The form field is required.  The label that was removed from `generalInformation.tsx` had the `required` class applied to it.

Reply:
Right but the property is not being set on the `DatePickerInput` here.
 
 INT1
 
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/124325935/ca371eca-6006-4509-9e8f-5a7efcf21da6)

this branch
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/124325935/9386f192-da06-408e-a553-14045768afd0)